### ISSUE TITLE
`bench` fails to build

### ISSUE THREAD

**Issue by h8liu** [dgraph-io/ristretto#34]
```
# github.com/dgraph-io/ristretto/bench
src/github.com/dgraph-io/ristretto/bench/bench.go:325:10: undefined: ristretto.PolicyLog
```

Found by gopkgs.io

**Comment by karlmcguire** [comment]
Thanks for the heads up. We just overhauled the hit ratio tracking in Ristretto so I'm going to have to decouple `bench` and add my own tracking. Give me a few hours.

**Comment by manishrjain** [comment]
Sorry for the breakage. Did the change late night and was more focused on the performance and coverage. Thanks Karl for taking care of it.

**Comment by karlmcguire** [comment]
Fixed in #35.
